components/ide/code/leeway.Dockerfile

```diff
@@ -47,6 +47,16 @@ RUN mkdir gp-code \
     && git fetch origin $GP_CODE_COMMIT \
     && git reset --hard FETCH_HEAD
 WORKDIR /gp-code
+ARG Brand=Gitpod
+ARG PrimaryColor=#FF8A00
+
+RUN sed -i -e "s/Gitpod/${Brand}/g" -e "8,11s/${Brand}/Gitpod/g" ./extensions/gitpod/package.nls.json
```

Comment:
There are certain gitpod links here for example gitpod Twitter, documentation, etc. So some entries are changed back to Gitpod